components/dot_menu/dot_menu.jsx

```diff
@@ -288,6 +291,37 @@ export default class DotMenu extends React.PureComponent {
                 );
             });
 
+        const pluginIntegrationsItems = this.props.pluginIntegrations.filter((item) => {
+            return item.location_type === PluginLocation.PLUGIN_LOCATION_POST_ACTION;
+        }).map((item) => {
+            return (
+                <Menu.ItemAction
+                    text={item.text}
+                    key={item.location_id + '_' + item.text}
```

Comment:
How unique is `item.location_id`? Is it possible for us to ever have a key collision here?

Reply:
We should make app id + location id to be unique.